### DISCUSSION TITLE
target url is working locally but after deployment the target server returns 417

### DISCUSSION THREAD

**Discussion by anonymouscatcher** [chimurai/http-proxy-middleware#737] [Q&A]
Hi There,

I'm running to some weird issues, I have setup a node server to use http proxy middleware and it's working locally but after deployment to another server it returns 417, 

```js
const proxy = createProxyMiddleware({
  logLevel: 'debug',
  target: process.env.XXXX,
  changeOrigin: true,
  onError(err, req, res, target) {
    console.log(err, target);
  }
});

export default proxy;
```

Anyone has any idea?

**Comment by anonymouscatcher** [comment]
To add more context on this: When I sent a request directly to from my server to target server it's working properly, but when I sent request from client to my server and from there to target server that doesn't work.

**Reply by chimurai** [reply]
You can try [v3@beta](https://github.com/chimurai/http-proxy-middleware/discussions/768).

You should be able to get better debug and log data.

```js
createProxyMiddleware({
   logger: console
})
```

```shell
# enable debug logging
DEBUG=http-proxy-middleware* node app.js
```
